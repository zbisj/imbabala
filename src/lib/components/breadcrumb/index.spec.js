import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Breadcrumb from ".";

describe("Breadcrumb", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <Breadcrumb
          items={[
            { link: "https://youtube.com/", label: "YouTube" },
            { link: "https://youtube.com/", label: "YouTube" },
          ]}
          variant="info"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
