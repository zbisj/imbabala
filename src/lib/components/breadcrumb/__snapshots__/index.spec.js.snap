// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb renders correctly 1`] = `
<ul
  className=""
>
  <li
    className=""
  >
    <a
      className=""
      href="https://youtube.com/"
    >
      YouTube
    </a>
    <span
      className=""
    >
      /
    </span>
  </li>
  <li
    className=""
  >
    <a
      className=""
      href="https://youtube.com/"
    >
      YouTube
    </a>
    <span
      className=""
    >
      /
    </span>
  </li>
</ul>
`;
